### ISSUE TITLE
Subtitle corpora

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/language-niche-research#1]
To do: 
- [x] Get relevant n-grams of the corpora. 
- [ ] Compare different n-grams for co-occurrence in both English and US corpora.
- [ ] Check out surprisal tool - used to be in NLTK. Find out why it was removed, where it can be used now. Get the package from the Piantadosi paper.

**Comment by RichardLitt** [comment]
Look at: 
- [ngrampy](https://github.com/piantado/ngrampy)
- This discussion of calculating perplexity in NLTK: [link](http://stackoverflow.com/questions/16509685/ngram-model-and-perplexity-in-nltk)
- This code -- potentially fixed -- but still not bundled with NLTK: [link](http://www.nltk.org/_modules/nltk/model/ngram.html)
- More python n-gram stuff [here](http://stackoverflow.com/questions/17531684/n-grams-in-python-four-five-six-grams)

**Comment by RichardLitt** [comment]
`cat`ing all files will result in strange behavior at the edges of the files. Would be best to cat each file as a new line in each file, and then parse them, maybe. Probably not a big deal.

**Comment by glupyan** [comment]
What kind of strange behavior? Just because it's a newline?

**Comment by RichardLitt** [comment]
Nah, I was worried that any ngrams that go over the divide wouldn't be useful as they would be from different speakers. Just checked again, and the corpus as a whole doesn't differentiate between speakers, though, so this is really moot. Catting it all is fine.
